scripts/update_inventory.py

```diff
@@ -0,0 +1,35 @@
+import docopt
+
+import teuthology
+import teuthology.config
+import teuthology.lock
+import teuthology.orchestra.remote
+
+import logging
+
+doc = """
+usage: teuthology-update-inventory -h
+       teuthology-update-inventory [-v] REMOTE [REMOTE ...]
+
+Update the given nodes' inventory information on the lock server
+
+
+  -h, --help            show this help message and exit
+  -v, --verbose         be more verbose
+  REMOTE                hostnames of machines whose information to update
+
+"""
+
+
+def main():
+    args = docopt.docopt(doc)
+    if args['--verbose']:
+        teuthology.log.setLevel(logging.DEBUG)
+
+    remotes = args.get('REMOTE')
+    for rem_name in remotes:
+        remote = teuthology.orchestra.remote.Remote(rem_name)
+        inventory_info = remote.inventory_info
+        del inventory_info['user']
```

Comment:
Ah, that turned out to be unnecessary. The user value would be incorrect, if paddles even used it. It doesn't yet, and when it does I'll have to fix it across the board. Removing that line.